.github/workflows/build-and-draft-release.yml

```diff
@@ -76,7 +76,10 @@ jobs:
       - uses: shogo82148/actions-upload-release-asset@v1
         with:
           upload_url: ${{ steps.release_drafter.outputs.upload_url }}
-          asset_path: azuredeploy.json
+          asset_path: |
+            azuredeploy.json
+            workbooks/Reliability Workbook.json
```

Comment:
Is this path right?
azuredeploy.json, Reliability Workbook.json and azuredeploy.bicep were all under workbooks folder. 

Also, update release draft seems skipped in the PR build. Is that expected?

Reply:
Thank you for your confirmation!

>Is this path right?
azuredeploy.json, Reliability Workbook.json and azuredeploy.bicep were all under workbooks folder.

Yes, this path is correct.
That's because `azuredeploy.json` is uploaded during previous task and downloaded to work folder on GitHub:
https://github.com/Azure/reliability-workbook/blob/9b7e31e80171f9c14e0d1f8470aa1da9bf75eda5/.github/workflows/build-and-draft-release.yml#L42-L46
https://github.com/Azure/reliability-workbook/blob/9b7e31e80171f9c14e0d1f8470aa1da9bf75eda5/.github/workflows/build-and-draft-release.yml#L72-L75

> Also, update release draft seems skipped in the PR build. Is that expected?

Yes. Even if PR build runs, release draft will not be created. I think it's better to create release only when push(merge) to main branch.